src/backend/opencl/kernel/iops.cl

```diff
@@ -7,13 +7,21 @@
  * http://arrayfire.com/licenses/BSD-3-Clause
  ********************************************************/
 
+#if CPLX
+inline bool is_nan(T in) { return (in.x != in.x) || (in.y != in.y); }
+#else
+inline bool is_nan(T in) { return (in != in); }
+#endif
+
 #if CPLX
 #define sabs(in) ((in.x)*(in.x) + (in.y)*(in.y))
 #ifdef MIN_OP
 void binOp(T *lhs, uint *lidx, T rhs, uint ridx)
 {
-    if ((sabs(lhs[0]) > sabs(rhs)) ||
-        (sabs(lhs[0]) == sabs(rhs) && *lidx < ridx)) {
+    if (((sabs(lhs[0]) > sabs(rhs)) ||
+         (sabs(lhs[0]) == sabs(rhs) &&
```

Comment:
Nit: I liked it better when the lidx < ridx condition was on the same line as the equality condition.

Reply:
:+1: Yeah that makes sense. I think I was just following suit with the other backends on how this condition was originally written. I could change the others too to this style.